ds_status_sync.py

```diff
@@ -308,7 +308,7 @@ def getstatus(self, stids: py.List[str]) -> py.List[object]:
         log.debug('getting status for %s:\ndata: %s\nheaders: %s',
                   stids, req.data, req.headers.items())
         s = self.__session
-        resp = s.send(s.prepare_request(req))  # type: ignore
```

Comment:
these were probably here for a reason.  please find out why and ensure its not broken

Reply:
These are used to ignore the type of the assigned value to a variable of a particular type
https://github.com/python/mypy/issues/8823

As the assigned value here is not type cased before assignment and it was also confirmed by:
https://github.com/kumc-bmi/ds-determined-tools/runs/6217641639?check_suite_focus=true

So tried commenting them but it didn't work https://github.com/kumc-bmi/ds-determined-tools/commit/aa95aff990d7accf10d92c640d7b067470d447df

The github still complained of pylint error

So removed it, but can place it back or revert it if you say

**commit:** ds_status_sync.py: pylint error: error: Unused "type: ignore" comment… · kumc-bmi/ds-determined-tools@aa95aff
… at 311,343,423 lines

Reply:
i get that getting rid of them fixes pylint but does it break mypy?
does github run mypy as well?